nokhwa-bindings-windows/src/lib.rs

```diff
@@ -966,7 +966,14 @@ pub mod wmf {
                     };
 
                     let frame_rate = match unsafe { media_type.GetUINT64(&MF_MT_FRAME_RATE) } {
-                        Ok(fps) => fps as u32,
+                        Ok(fps) => {
+                            let mut numerator = (fps >> 32) as u32;
+                            let denominator = fps as u32;
+                            if denominator != 1 {
+                                numerator = 0;
+                            }
+                            numerator
```

Comment:
As fare as I understand the API https://learn.microsoft.com/en-us/windows/win32/medfound/mf-mt-frame-rate-attribute the numerator should be divided by the denominator to get the frame rate.

```suggestion
                            if denominator != 0 {
                                numerator / denominator
                            } else {
                                0
                            }
```

Reply:
I think a rounded framerate is better than no frame rate